kairon/api/app/routers/auth.py

```
@@ -164,13 +165,27 @@ async def sso_callback(
+    UserActivityLogger.add_log(a_type=UserActivityType.sso_login.value, email=user_details['email'],
+                               data={"username": user_details['email'], "sso_type": sso_type})
     return {
         "data": {"access_token": access_token, "token_type": "bearer"},
         "message": """It is your responsibility to keep the token secret.
         If leaked then other may have access to your system.""",
     }
 
 
+@router.post("/logout", response_model=Response)
+async def logout(
+        current_user: User = Depends(Authentication.get_current_user)
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)